src/Badge.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+class Badge extends React.Component {
+  render() {
+    let classes = ['badge'];
+    if (this.props.text) {
+      classes.push(this.props.text);
```

Comment:
Please use `classnames` (https://github.com/JedWatson/classnames) library to compose class names.
I am not sure if the text could be other than `new`, in that case it should be a boolean prop.